libs/wingc/src/parser.rs

```diff
@@ -332,6 +332,43 @@ impl<'s> Parser<'s> {
 		})
 	}
 
+	fn build_struct_definition_statement(&self, statement_node: &Node) -> DiagnosticResult<StmtKind> {
+		let name = self.node_symbol(
+			&statement_node
+				.child_by_field_name("name")
+				.expect("struct definitions should always have a name"),
```

Comment:
nit: There's a helper method that can help avoid the expect

```suggestion
			&self.get_child_field(&statement_node, "name")?
```

Reply:
That is fucking wonderful! Thanks for the nit!